rootfs/api/models/build.py

```diff
@@ -52,8 +52,8 @@ def version(self):
         return 'git-{}'.format(self.sha) if self.source_based else 'latest'
 
     def create(self, user, *args, **kwargs):
-        latest_release = self.app.release_set.latest()
-
+        latest_release = self.app.release_set.filter(failed=False).latest()
```

Comment:
Ah now I get it. I was completely missing that context as there is no documentation to rely on here. In the future, documenting that change as well as the justification in the original comment of the PR would help clarify any confusion. :)